draft-ietf-mls-architecture.md

```diff
@@ -622,8 +611,9 @@ use them for tracking.
 > Prefer using encrypted group operation messages to avoid privacy
 > issues related to non-encrypted signatures.
 
-Note that in the default case of encrypted handshake messages, the
-application level must make sure that the access control policies are
+Note that in the default case of encrypted handshake messages, any access
+control policies will be applied at the clien, so the
```

Comment:
```suggestion
control policies will be applied at the client, so the
```
Maybe mention here that an extension is a great way of making sure all members agree on something? 🙂